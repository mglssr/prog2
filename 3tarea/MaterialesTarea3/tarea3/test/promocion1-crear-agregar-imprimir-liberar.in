crearPromocion 01/01/2001 01/06/2001 10 1
imprimirPromocion
liberarPromocion
crearPromocion 01/01/2002 01/06/2002 10 1
agregarAPromocion 5 0 1 2 3 4
imprimirPromocion
agregarAPromocion 5 5 6 7 8 9
imprimirPromocion
agregarAPromocion 5 10 11 12 13 14
imprimirPromocion
liberarPromocion
Fin
