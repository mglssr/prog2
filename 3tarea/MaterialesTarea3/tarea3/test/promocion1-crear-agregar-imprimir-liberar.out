Bienvenido al programa principal de MercadoFinger. Por favor ingrese su comando:
1> Promocion creada con exito.
2> Promocion #1 del 1/1/2001 al 1/6/2001
Productos: 
3> Promocion liberada con exito.
4> Promocion creada con exito.
5> Productos agregados a la promocion de forma exitosa.
6> Promocion #1 del 1/1/2002 al 1/6/2002
Productos: 0 1 2 3 4 
7> Productos agregados a la promocion de forma exitosa.
8> Promocion #1 del 1/1/2002 al 1/6/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
9> Productos agregados a la promocion de forma exitosa.
10> Promocion #1 del 1/1/2002 al 1/6/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
11> Promocion liberada con exito.
12> Fin.
